test/e2e/configurer/factory.go

```diff
@@ -212,11 +218,38 @@ func NewSoftwareUpgradeConfigurer(t *testing.T, isDebugLogEnabled bool, upgradeP
 	return NewUpgradeConfigurer(t,
 		[]*chain.Config{
 			// we only need 1 chain for testing upgrade
-			chain.New(t, containerManager, initialization.ChainAID, validatorConfigsChainA, nil),
+			chain.New(t, containerManager, initialization.ChainAID, updateNodeConfigNameWithIdentifier(validatorConfigsChainA, identifier), nil),
 		},
 		withUpgrade(baseSetup), // base set up with upgrade
 		containerManager,
 		upgradePath,
 		0,
 	), nil
 }
+
+func identifierName(t *testing.T) string {
```

Comment:
Why is this needed? Just curious.

Reply:
If you would run two or more tests at the same time they might match the name of the container and mess up everything :exploding_head: